### PR TITLE
Add PlayStatus

### PR DESCRIPTION
**Pull request by okb12**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/playstatus.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'playstatus' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://github.com/nbolar/PlayStatus/releases/latest/download/PlayStatus.app.zip'
```
Comment (vitorgalvao) [inline Casks/playstatus.rb:5]: Missing [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/playstatus.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'playstatus' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/playstatus.rb:2]: Needs version.

File: Casks/playstatus.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/playstatus.rb:3]: Needs checksum.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> + The Cask is too obscure. Examples:
>   + A self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 30 forks, 30 watchers, and 75 stars).
>   + [Electronic Identification (eID) software](https://github.com/Homebrew/homebrew-cask/issues/59021).

**Comment by vitorgalvao** [comment]
@SMillerDev Isn’t the notability check for casks live yet? I was under the impression it was. It it is, something went wrong, as it didn’t fire here.

**Comment by SMillerDev** [comment]
It is, I'll check what's happening.

**Comment by SMillerDev** [comment]
I think CI needs to be adjusted to run `brew cask audit --new-cask <token>` on new casks. Otherwise the notability check isn't triggered.

**Comment by ran-dall** [comment]
> I think CI needs to be adjusted to run brew cask audit --new-cask <token> on new casks. Otherwise the notability check isn't triggered.

@SMillerDev @vitorgalvao I'll see what I can do.

**Pull request closed without merging**